src/api/v1/gitea/endpoints/get_repo.rs

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: 2023 Gergely Nagy
+// SPDX-FileContributor: Gergely Nagy
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use axum::{
+    body::Body,
+    extract::Path,
+    http::{Request, StatusCode},
+    response::{IntoResponse, Redirect},
+};
+
+#[allow(unused)]
+use log::{debug, error, info, trace, warn};
+
+use super::super::utils::gitea_api_get_latest_tag_url;
+
+pub async fn get_repo(
+    Path((host, user, repo)): Path<(String, String, String)>,
+    request: Request<Body>,
+) -> impl IntoResponse {
+    if repo.ends_with(".tar.gz") {
+        let latest_tag_url = gitea_api_get_latest_tag_url(
+            host.clone(),
+            user.clone(),
+            repo.clone()
+                .strip_suffix(".tar.gz")
+                .expect("couldn't strip .tar.gz suffix")
+                .to_string(),
+        )
+        .await
+        .expect("failed to await gitea_api_get_latest_tag_url");
+        trace!("{latest_tag_url:#?}");
```

Comment:
I also might have done this elsewhere, but I think it's better if we add a minimal context to this:
```suggestion
        trace!("latest_tag_url: {latest_tag_url:#?}");
```